### ISSUE TITLE
course-info.md Typo(?)

### ISSUE THREAD

**Issue by surbhipatel98** [laboon/CS1632_Fall2018#27]
"Why's (Poignant) Guide to Ruby by _why the lucky stiff."
Is that underscore supposed to be there? I believe his pseudonym is just "why the lucky stiff"

-Surbhi Patel

**Comment by laboon** [comment]
Nope, the underscore is supposed to be there.  It was often elided over but it is technically part of his full alias.
